### PR TITLE
chore: bump friendsofphp/php-cs-fixer from 2 => 3

### PR DESCRIPTION
**Pull request by Chris53897**
Outdated: bump php from >=7.2 to >=7.2.5 for this
Master changed to ` "php": ">=8.0",` meanwhile.

### CODE REVIEW

**Comment by Chris53897** [comment]
https://github.com/omines/datatables-bundle/pull/228

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -11,12 +21,13 @@
 $finder = PhpCsFixer\Finder::create()
```
Comment (maxhelias) [inline .php-cs-fixer.php:21]: It seems to me that the `Finder::create` is also deprecated

Comment (Chris53897) [reply .php-cs-fixer.php:21]: I could not find infos about that. 
PhpCsFixer extends the symfony finder.
https://github.com/symfony/finder/blob/6.0/Finder.php#L76

Can you please give some hints about the deprecation?

Comment (maxhelias) [reply .php-cs-fixer.php:21]: Not found either, bad memory sorry 😅

**Review by maxhelias — APPROVED** [review]

**Comment by Chris53897** [comment]
Version 2 of this is blocking some symfony6 components from installing in dev environment. (Example: symfony/options-resolver)
https://packagist.org/packages/friendsofphp/php-cs-fixer#v2.19.3

**Pull request closed without merging**